### DISCUSSION TITLE
Lost the ability to create a corresponding relationship after 9.0.0-rc.100 upgrade

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#9469] [Archive]
Hi everyone, 

The UI to create a new field has changed after upgrading to rc.100, when creating a relationship there was an option to create a corresponding relationship in another Model, wonder if that is still possible somehow? 

In my personal opinion, I liked the old UI much better, a list is better than dozen confusing icons.

**Comment by jamescammarano** [comment]
> In my personal opinion, I liked the old UI much better, a list is better than dozen confusing icons.

I was thinking about this and I was wondering if having a filter bar at the top would make it easier to find things.

> The UI to create a new field has changed after upgrading to rc.100, when creating a relationship there was an option to create a corresponding relationship in another Model, wonder if that is still possible somehow?

What kind of relationship were you trying to make?

**Reply by DeanKamali** [reply]
I'm trying to do `one to many relationship` why can't I reference another collection by Id? "grayed out" 

![Screen Shot 2021-11-05 at 9 18 05 AM](https://user-images.githubusercontent.com/1252959/140516442-6111f987-efad-48ab-a2a2-69a9c94d55d5.png)

**Comment by benhaynes** [comment]
You still have the option to use the previous (Advanced) Field Flow if you prefer... it's a link directly below the big Create Field button. This option still has the "Create Corresponding Field" etc. 👍

**Reply by DeanKamali** [reply]
it was an oversight on my end and didn't see that option, thanks for pointing it out.
